recipes/linux-sysroot/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set build_number = "0" %}
+package:
+  name: linux-sysroot
+  version: 2.12
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/glibc-2.12-1.212.el6.x86_64.rpm
+    sha256: b893707c0dc87a556274feb53c544515a1d3b0fe2743c454dece682f5b7d5719
+    folder: binary
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/glibc-common-2.12-1.212.el6.x86_64.rpm
+    sha256: 721dfba71f7402d94070fe155f468666f2b785d56a638eec3d6ba3dc97fbef75
+    folder: binary-glibc-common
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/glibc-devel-2.12-1.212.el6.x86_64.rpm
+    sha256: 65b97f1f868afc090b1beec846490d1e6974751f188fd42b1ebc59d1c2866549
+    folder: binary-glibc-devel
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/tzdata-2018e-3.el6.noarch.rpm
+    sha256: d66cde6ab07be8739b2222c0463cc7c0218a55f1f825944545cef692107ae685
+    folder: binary-tzdata
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/glibc-headers-2.12-1.212.el6.x86_64.rpm
+    sha256: 7f1f3a2cf562465f2c18031e4e76734987ea8489c1202301e019dfb722ff5031
+    folder: binary-glibc-headers
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/kernel-headers-2.6.32-754.el6.x86_64.rpm
+    sha256: 93b20d547637e3d25a9604b8034334afa910f61898da0f265a3969a68bf8cc6d
+    folder: binary-kernel-headers
+
+build:
+  number: {{ build_number }}
+  noarch: generic
```

Comment:
Is this typical with CDTs? Where do I find the relevant docs for this?

Reply:
This is how they do it, it is really strange though.

I think the idea is that: "this is just a hack, so might as well leave it in noarch"

Reply:
> I think the idea is that: "this is just a hack, so might as well leave it in noarch"

No, it's because these CDTs are installed in `build` for the compiler to find, but really belong in `host`. To enable cross-compiling they need to be noarch:generic.